threadit-api-tests/ServicesTests/SpoolServiceTests.cs

```diff
@@ -0,0 +1,61 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Metadata.Internal;
+using ThreaditAPI.Database;
+using ThreaditAPI.Models;
+using ThreaditAPI.Repositories;
+using ThreaditAPI.Services;
+
+namespace ThreaditTests.Repositories;
+
+public class SpoolServiceTests
+{
+    private SpoolService _spoolService;
+    private PostgresDbContext _dbContext;
+
+    [SetUp]
+    public void Setup()
+    {
+        _dbContext = new PostgresDbContext();
+        _dbContext.Database.EnsureDeleted();
+        _dbContext.Database.Migrate();
+        _spoolService = new SpoolService(_dbContext);
+    }
+
+    [TearDown]
+    public void Cleanup()
+    {
+        _dbContext.Database.EnsureDeleted();
+    }
+
+    [Test]
+    public void RetrieveSpoolByName_NotExists_ShouldFail()
+    {
+        Assert.ThrowsAsync<Exception>(async () => await _spoolService.GetSpoolByNameAsync("doesNotExistSpool"));
```

Comment:
I didnt know this was a thing. we should change UserService tests to do the same now so we have one standard

Reply:
in another PR 👍